cfme/tests/containers/test_blacklisted_container_events.py

```diff
@@ -69,6 +68,18 @@
 }
 
 
+def restore_advanced_settings(appliance):
```

Comment:
Will `update_advanced_settings` be modified in the future to behave in this way? Or is this a temporary workaround?

Reply:
In what way are you referring to here?

Reply:
Nevermind, I see that here you are only changing a single setting that depends on the version of the appliance.